README.md

```
@@ -1,69 +1,308 @@
-# Please read!
+# 🔍 SherlockRamosBot
 
+<div align="center">
 
-**For any problems running this specific bot:** [Discord Project Post](https://discord.com/channels/974519864045756446/1055336272543092757)
+**ChatBot inteligente no Discord para auxiliar concurseiros brasileiros com dúvidas jurídicas**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)